### PR TITLE
fix bug with FromParams

### PR DESCRIPTION
**Pull request by epwalsh**
Fixes #

Changes proposed in this pull request:

-

## Before submitting

- [ ] I've read and followed all steps in the [Making a pull request](https://github.com/allenai/tango/blob/main/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [ ] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/allenai/tango/blob/main/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [ ] If this PR fixes a bug, I've added a test that will fail without my fix.
- [ ] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

## After submitting

- [ ] All GitHub Actions jobs for my pull request have passed.

### CODE REVIEW

File: tango/common/from_params.py [hunk 1]
```diff
@@ -421,8 +421,6 @@ def construct_arg(
         if origin is None and isinstance(popped_params, annotation):
             return popped_params
         elif popped_params is not None:
-            subextras = create_extras(annotation, extras)
```
Comment (dirkgr) [inline tango/common/from_params.py:424]: Can we get rid of `subextras` everywhere then?

**Review by dirkgr — APPROVED** [review]

File: tango/common/from_params.py (hunk 1 above)
Comment (epwalsh) [reply tango/common/from_params.py:424]: Done

**Review by dirkgr — APPROVED** [review]
I'm not sure if I should be happy or scared.

**Comment by epwalsh** [comment]
Scared, definitely scared.

**Pull request merged**